practical-unit-testing-with-rtl/Exercise2/solution/src/components/Score/Score.module.css

```diff
@@ -0,0 +1,33 @@
+.scoreContainer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.scoreText {
+  font-size: 32px;
```

Comment:
In general, I would use `rem` instead of `px` for font sizes. I’m guessing this is about `2rem`.